src/ut/baseresolver_test.cpp

```diff
@@ -95,6 +98,14 @@ class BaseResolverTest : public ResolverTest
     return targets;
   }
 
+  /// Implements the resolve_iter mathod using a BaseResolver
```

Comment:
This is just a helper method, right? The comment makes it sound like you are implementing an interface but I don't think that's the case.